analytics/scala/src/main/scala/com/koverse/example/spark/WordCountDatasetTransform.scala

```diff
@@ -16,15 +16,57 @@
 
 package com.koverse.example.spark
 
+import java.io.Serializable
+
 import com.koverse.sdk.Version
 import com.koverse.sdk.data.Parameter
 import com.koverse.sdk.transform.scala.{DatasetTransform, DatasetTransformContext}
 import org.apache.spark.sql.Dataset
 
+case class Message(var article: String, var title: String, var id: Long) extends Serializable {
```

Comment:
Scala case classes extend Serializable by default. You shouldn't need that there.

Reply:
Ok, I will remove